includes/admin/class-wc-rest-stripe-agentic-commerce-controller.php

```
@@ -0,0 +1,254 @@
+	 */
+	public function trigger_sync() {
+		if ( ! class_exists( 'WC_Stripe_Agentic_Commerce_Integration' ) ) {
+			return new WP_Error(
+				'stripe_agentic_commerce_unavailable',
+				__( 'Agentic Commerce integration is not available.', 'woocommerce-gateway-stripe' ),
+				[ 'status' => 503 ]
+			);
+		}
+
+		try {
+			$integration = new WC_Stripe_Agentic_Commerce_Integration();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '129', column '23'). (undefined)

(MissingImport)